ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { FILE_FIELD_NAME, PROJECT_DESC_FIELDS, FAMILY_FIELD_ID, INDIVIDUAL_FIELD_ID } from 'shared/utils/constants'
+import { BaseBulkContent, BASE_UPLOAD_FORMATS } from 'pages/Project/components/edit-families-and-individuals/BulkEditForm'
+import { INDIVIDUAL_CORE_EXPORT_DATA, INDIVIDUAL_ID_EXPORT_DATA } from 'pages/Project/constants'
```

Comment:
Our general policy is to not import component and constants in between different `pages` folders, and we instead move them to the `shared` folder. However, that adds a bunch of stuff to the diff and makes the PR harder to read. I thin what makes sense is to leave this as is for now, and then immediately after this PR is merged put up a second PR to move these. I would say `BaseBulkContent` would move to its own file in `shared/components/form` and the constant can just move to the main `shared/utils/constants`

Reply:
Okay.

Reply:
reminder that once this PR is merged to dev you should put up another PR  to move the necessary components and constants to the shared folder. Don't move the whole files, just the shared parts